src/cascadia/TerminalControl/TermControlAutomationPeer.cpp

```diff
@@ -170,6 +177,23 @@ namespace winrt::Microsoft::Terminal::TerminalControl::implementation
         };
     }
 
+    HRESULT TermControlAutomationPeer::GetHostUiaProvider(IRawElementProviderSimple** provider)
+    {
+        return S_OK;
+    }
+
+    RECT TermControlAutomationPeer::GetPadding()
+    {
+        auto padding = _termControl->GetPadding();
+        return {
+            gsl::narrow<LONG>(padding.Left),

```

Comment:
probably safe to narrow_cast these -- i _think_ that's the nonthrowing version

Reply:
correct, narrow_cast is the non-throwing one.